src/modules/dbus/dbusmodule.cpp

```diff
@@ -271,6 +271,47 @@ class Controller1 : public ObjectVTable<Controller1> {
         return instance_->inputMethodManager().currentGroup().name();
     }
 
+    dbus::Variant
+    inputMethoEntry2Variant(const InputMethodEntry &inputMethodEntry) {
```

Comment:
the existing field are commonly used and are not optional field, so you should not put them into the variant map.